client/src/pages/Initial/orientation-website.code-workspace

```diff
@@ -0,0 +1,11 @@
+{
+	"folders": [
+		{
```

Comment:
not sure what this file is doing here? might've been something that was added accidentally?
make sure to uncommit this as well if it is